src/shared/pages/list/entityList/__tests__/entityListView.spec.tsx

```diff
@@ -78,11 +78,13 @@ beforeEach(() => {
         entities: [{
             simpleName: 'testSimpleName1',
             uuid: 'testId1',
-            title: 'testTitle1'
+            title: 'testTitle1',
+            active: true
         }, {
             simpleName: 'testSimpleName2',
             uuid: 'testId2',
-            title: 'testTitle2'
+            title: 'testTitle2',
+            active: true
```

Comment:
For this test it would be useful to have both true and false included when rendering the snapshot.

```suggestion
            active: false
```

Remember to re-run tests locally to update the snapshot too :)